guides/pricing_and_taxation.md

```diff
@@ -0,0 +1,127 @@
+
+***
+Taxation within the United States can get exceptionally complex, with different states, counties and even cities having different taxation rates. If you are shipping interstate within the United States, we would strongly advise you to use the [Solidus Avatax](https://github.com/solidusio/solidus_avatax) extension so that you get correct tax rates.
+***
+
+### Shipping vs. Billing Address
+
+Most tax jurisdictions base the tax on the shipping address of where the order is being shipped to. So in these cases the shipping address is used when determining the tax zone. Solidus does, however, allow you to use the billing address to determine the zone.
+
+To determine tax based on billing address instead of shipping address you will need to set the `Spree::Config[:tax_using_ship_address]` preference to `false`.
+
+### Calculators
+
+Every `Spree::TaxRate` has a `Spree::Calculator` that calculates the correct tax amout for a given shipment or line item. In most cases you should be able to use Solidus's `DefaultTax` calculator. It is suitable for both sales tax and price-inclusive tax scenarios.
+
+To learn more about tax calculators, have a look at the [Default Tax Calculator Specs](https://github.com/solidusio/solidus/blob/master/core/spec/models/spree/calculator/default_tax_spec.rb) or its [implementation](https://github.com/solidusio/solidus/blob/master/core/app/models/spree/calculator/default_tax.rb).
+
+## Pricing
+
+In VAT-style jurisdictions, taxes have to be included in the price shown to customers (in index and product display pages as well as when adding them to the cart). In order to configure this, you need to first configure your tax rates, categories, and zones, and then update your products and variants with the "Rebuild VAT prices" checkbox checked. Solidus will then proceed to calculate the correct net price and create prices for all countries with `included_in_price` VAT rates. It will also generate a fallback "export" price, where the prices `country_iso` is `nil` (not set). New products will behave as if that checkbox is checked. You can adjust the prices in the "Prices" tab if you need to.
+
+Solidus can handle several prices for the same country, and will always select the most recently updated price.
+
+## Prices in the backend
+
+In the Solidus admin backend, prices will be shown and entered with or without VAT, depending on the configuration value `Spree::Config.admin_vat_country_iso`. This has to be set to the two-letter ISO code of the country your admin thinks in. For a store that mostly sells to Germany, this would be set to `DE`. The effect of this is that now all prices in the backend will assume to have the German VAT rate included.
```

Comment:
I believe there are open issues which talk about adding more robust pricing management in the admin. Perhaps more links to those issues to suggest further active development?